react/ProductHighlights.tsx

```diff
@@ -36,7 +36,7 @@ interface ProductHighlightContextProviderProps {
 
 function createFilterHighlight(filter: Filter) {
   return function filterHighlight(highlight: Highlight) {
-    const hasHighlight = filter.highlightNames.includes(highlight.name)
+    const hasHighlight = filter.highlightNames?.includes(highlight.name)
```

Comment:
🪙🪙  If this is needed, then the prop might be also optional. Since it is currently required, someone could easily remove it in the future, assuming it would not be possible for `highlightNames` to be undefined.

Reply:
Yes, and checking the [documentation](https://github.com/vtex-apps/product-highlights#product-highlights-props), I'm not sure it makes sense to not receive this prop on `filter`, because otherwise why is the client using the filter object if it's not filtering anything?

I think the issue we are trying to solve on that slack thread is more related to some configuration problem.